### PR TITLE
Add MODIS Terra/Aqua satwinds assimilation to end-to-end GDASApp validation

### PR DESCRIPTION
**Pull request by BrettHoover-NOAA**
Adding satwinds from the Moderate Resolution Imaging Spectroradiometer (MODIS) from Terra/Aqua to GDASApp end-to-end testing

new files include:
parm/atm/obs/config/satwind_modis_terra.yaml.j2: QC filter YAML for MODIS Terra satwinds (jinja2 standard)
parm/atm/obs/config/satwind_modis_aqua.yaml.j2: QC filter YAML for MODIS Aqua satwinds (jinja2 standard)
parm/ioda/bufr2ioda/bufr2ioda_satwind_amv_modis.json: JSON containing data format, sensor, and satellite information for MODIS Terra/Aqua satwinds
ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_modis.py: bufr2ioda code for extracting MODIS Terra/Aqua satwinds from BUFR

See #1029 for end-to-end testing results. Acceptance and ob-errors agree well with some expected deviation. Thinning is currently turned off in both GSI and JEDI for these tests.

Note: We are still using `qualityInformationWithoutForecast` as the variable-name for QI in the IODA converter. This variable-name is currently not registered in the IODA ObsSpace.yaml, but there is an ongoing discussion with JCSDA to have it added, issue is here: https://github.com/JCSDA-internal/ioda/issues/1233

### CODE REVIEW

File: ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_modis.py [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+                .write_attr('long_name', 'Observation Type based on Satellite-derived Wind Computation Method and Spectral Band') \
+                .write_data(obstype2)
+
+            # ObsType based on computation method/spectral band
+            obsspace.create_var('ObsType/windNorthward', dtype=obstype2.dtype, fillval=swcm2.fill_value) \
+                .write_attr('long_name', 'Observation Type based on Satellite-derived Wind Computation Method and Spectral Band') \
+                .write_data(obstype2)
+
+            # Pressure
+            obsspace.create_var('MetaData/pressure', dtype=pressure2.dtype, fillval=pressure2.fill_value) \
```
Comment (nicholasesposito) [inline ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_modis.py:402]: Suggest moving all of the MetaData variables to be together.

**Review by nicholasesposito — COMMENTED** [review]
One small suggestion in the python script. Otherwise, looks fine to me

File: ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_modis.py (hunk 1 above)
Comment (BrettHoover-NOAA) [reply ush/ioda/bufr2ioda/bufr2ioda_satwind_amv_modis.py:402]: Should be addressed in https://github.com/NOAA-EMC/GDASApp/pull/1030/commits/2323ec91ac96f49ac3e5877c3c191c33041829ad

**Comment by emilyhcliu** [comment]
@BrettHoover-NOAA I will check out the branch and run tests!  The end-to-end results you reported look good!

**Comment by BrettHoover-NOAA** [comment]
@emilyhcliu can this PR get reviewed and merged? I am going to go back and make a change to all of the satwind code to change the variable-name for `qualityInformationWithoutForecast` to comply with ioda ObsSpace.yaml standards, but this should get in first.

**Review by PraveenKumar-NOAA — APPROVED** [review]
Looks good to me.

**Review by emilyhcliu — APPROVED** [review]

**Pull request merged**